contracts/truefi2/LoanToken2.sol

```diff
@@ -137,11 +147,12 @@ contract LoanToken2 is ILoanToken2, ERC20 {
         uint256 _amount,
         uint256 _term,
         uint256 _apy
-    ) public ERC20("TrueFi Loan Token", "LOAN") {
+    ) public {
         require(_lender != address(0), "LoanToken2: Lender is not set");
 
         pool = _pool;
         token = _pool.token();
+        ERC20.__ERC20_initialize(concat("TrueFi ", token.name()), concat("tf", token.symbol()));
```

Comment:
Does it not create some confusion?  In this case loan tokens would have same names and symbols as their corresponding pools.